gcp_variant_transforms/transforms/merge_headers.py

```diff
@@ -26,16 +26,16 @@ class _MergeHeadersFn(beam.CombineFn):
   def create_accumulator(self):
     return vcf_header_io.VcfHeader()
 
-  def add_input(self, source, to_merge):
+  def add_input(self, source, to_merge, *args, **kwargs):
```

Comment:
Please also remove args/kwargs from here. This is also not a common convention (e.g. https://beam.apache.org/documentation/programming-guide/#advanced-combinations-using-combinefn)

Reply:
Done